storage/rocksdb/ha_rocksdb.cc

```diff
@@ -12456,16 +12463,24 @@ int ha_rocksdb::update_write_sk(const TABLE *const table_arg,
       rc = check_partial_index_prefix(table_arg, kd, row_info.tx,
                                       row_info.old_data);
       if (!rc) {
-        row_info.tx->get_indexed_write_batch(m_tbl_def->get_table_type())
-            ->SingleDelete(kd.get_cf(), old_key_slice);
+        const auto s =
+            row_info.tx->get_indexed_write_batch(m_tbl_def->get_table_type())
+                ->SingleDelete(kd.get_cf(), old_key_slice);
+        if (!s.ok()) {
+          return rdb_error_to_mysql(s);
```

Comment:
nit: in ha_rocksdb::delete_row(), if SD or delete fail, it will call tx->set_status_error() instead of rdb_error_to_mysql(). Although these two function are similar, maybe we should have same behavior?